src/App.js

```diff
@@ -1,20 +0,0 @@
-import React from 'react';
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
-import Nav from './components/Nav';
-import Books from './components/Books';
-import './App.css';
-import Categories from './components/Categories';
-
-const App = () => (
```

Comment:
Great job @elad237 🤝

- [ ] I am getting the error below whenever I try to run the server for your project. Kindly fix this error and resubmit your project for re-review.
![error](https://user-images.githubusercontent.com/63917725/224519386-5c0095a6-261e-4f6b-a97b-a7f643b99ac6.PNG)

Reply:
Thanks or the review but everything is working well on my side

> Great job @elad237 🤝
> 
> * [ ]  I am getting the error below whenever I try to run the server for your project. Kindly fix this error and resubmit your project for re-review.
>   ![error](https://user-images.githubusercontent.com/63917725/224519386-5c0095a6-261e-4f6b-a97b-a7f643b99ac6.PNG)